src/main/resources/db/changelog/changes/14-create-orders-table.yaml

```diff
@@ -0,0 +1,63 @@
+databaseChangeLog:
+  - changeSet:
+      id: create_orders_table
+      author: Pavlo
+      changes:
+        - createTable:
+            tableName: orders
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: user_id
+                  type: BIGINT
+                  constraints:
+                    notNull: true
+              - column:
+                  name: status
+                  type: enum ('PENDING','PROCESSING','SHIPPED','DELIVERED','CANCELLED')
+                  defaultValue: 'PENDING'
+                  constraints:
+                    notNull: true
+              - column:
+                  name: total
+                  type: decimal
+                  constraints:
+                    notNull: true
+              - column:
+                  name: order_date
+                  type: TIMESTAMP
+                  constraints:
+                    notNull: true
+              - column:
+                  name: shipping_address
+                  type: VARCHAR(255)
+                  constraints:
+                    notNull: true
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
+              - column:
+                  name: order_item_id
+                  type: bigint
+              - addForeignKeyConstraint:
+                  baseTableName: orders
+                  baseColumnNames: user_id
+                  referencedTableName: users
+                  referencedColumnNames: id
+                  constraintName: fk_order_user
+                  onDelete: CASCADE
+              - addForeignKeyConstraint:
+                  baseTableName: orders
+                  baseColumnNames: order_item_id
+                  referencedTableName: order_items
+                  referencedColumnNames: id
+                  constraintName: fk_order_item_order
+                  onDelete: CASCADE
```

Comment:
Why do you duplicate addForeignKeyConstraint?

Reply:
I thought that i need it because first one was for user and second one is for order_item

The same as prev